examples/config.yml

```diff
@@ -1,26 +1,28 @@
 ---
-metrics:
-- name: example_global_value
-  path: "{ .counter }"
-  help: Example of a top-level global value scrape in the json
-  labels:
-    environment: beta # static label
-    location: "planet-{.location}"          # dynamic label
+modules:
+  default:
+    metrics:
+    - name: example_global_value
+      path: "{ .counter }"
+      help: Example of a top-level global value scrape in the json
+      labels:
+        environment: beta # static label
+        location: "planet-{.location}"          # dynamic label
 
-- name: example_value
-  type: object
-  help: Example of sub-level value scrapes from a json
-  path: '{.values[?(@.state == "ACTIVE")]}'
-  labels:
-    environment: beta # static label
-    id: '{.id}'          # dynamic label
-  values:
-    active: 1      # static value
-    count: '{.count}' # dynamic value
-    boolean: '{.some_boolean}'
+    - name: example_value
+      type: object
+      help: Example of sub-level value scrapes from a json
+      path: '{.values[?(@.state == "ACTIVE")]}'
+      labels:
+        environment: beta # static label
+        id: '{.id}'          # dynamic label
+      values:
+        active: 1      # static value
+        count: '{.count}' # dynamic value
+        boolean: '{.some_boolean}'
+    headers:
+      X-Dummy: my-test-header
 
-headers:
-  X-Dummy: my-test-header
 
 # If 'body' is set, it will be sent by the exporter as the body content in the scrape request. The HTTP method will also be set as 'POST' in this case.
```

Comment:
Should we also indent these comments since they're now part of the default module?

Reply:
Updated.